### PR TITLE
Refactor/connect auth with backend

### PR DESCRIPTION
**Pull request by ahmedasad236**

### CODE REVIEW

**Review by a-asmaa — CHANGES_REQUESTED** [review]
@ahmedasad236 please remove apps that are created for test

**Comment by ahmedasad236** [comment]
Done @a-asmaa

**Review by a-asmaa — CHANGES_REQUESTED** [review]
there're some comments that need to be checked through different components "generic", 
like:
- Delete wrapping box and tag number and use their style.
- sending form data as params in URL
- Move backend integration "fetch" into a separate file
- ....
and please change create integration in sidebar => to integration and should route to the integration list page

File: packages/web/src/components/AuthNewForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                        {...formUtilities.register(`fields.${lastIndex}.description`)}
+
+                    />
+
+                    <FormGroup sx={{display: 'flex', flexDirection: 'row', justifyContent: 'space-between'}}>
+                        <FormControlLabel
+                            sx={{ my: '0.5rem' }}
+                            control={<Checkbox {...formUtilities.register(`fields.${lastIndex}.required`)} />}
+                            label="Required"
+                        />
```
[the remark this answers is not in the document]
Comment (ahmedasad236) [reply packages/web/src/components/AuthNewForm/index.tsx:68]: **YES**

File: packages/web/src/components/AuthSecondStep/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+function AuthSecondStep({control, register}: AuthStepsProps) {
+    return (
+        <CustomAccordion tag={<TagNumber text="Step 2" />} heading="Configure a Test Request">
+
+            <WrappingBox>
+                Add a simple API endpoint to test user credentials. Also provide the required headers for this endpoint.
+            </WrappingBox>
+
+            <AuthTestRequesForm control={control} register={register} />
```
[the remark this answers is not in the document]
Comment (ahmedasad236) [reply packages/web/src/components/AuthSecondStep/index.tsx:18]: ** I tried to do that, but I think it looks good. as the form content will make this component much bigger**

**Review by a-asmaa — APPROVED** [review]

**Pull request merged**